adapters/adview/adview.go

```diff
@@ -100,7 +100,8 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
-	bidResponse.Currency = response.Cur
+	bidResponse.Currency = "USD" //we just support USD for resp
```

Comment:
@AdviewOpen Should the `req.Cur` in the request being sent to your exchange needs to be updated as well if your exchange only supports USD? Currently, `req.Cur` would be set to whatever is specified by the publisher. What does your exchange do if the requested currency in `req.Cur` is not USD?

Reply:
thx and i changed codes and add test script ,pls check it .